src/System.Management.Automation/engine/CommonCommandParameters.cs

```diff
@@ -232,6 +232,22 @@ public string PipelineVariable
             set { _commandRuntime.PipelineVariable = value; }
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG.md.https://dependabot.com should work
```powershell
retry of URL failed with error: The SSL connection could not be established, see inner exception.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```